db/migrate/20230126164114_create_faculty_member.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFacultyMember < ActiveRecord::Migration[6.0]
+  def change
+    create_table :faculty_members do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :webaccess_id, null: false, default: ""
```

Comment:
We still need the `middle_name` and `department` fields added to this.  Also, we should remove the `default: ""` on line 6.  We'll want first and last name to be `null: false` as well.